### PR TITLE
Adds Docker Compose and Kubernetes Helm Chart

### PR DESCRIPTION
**Pull request by JBAhire**
- Added docker-compose deployment option
- Added Helm charts repository
- updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,37 @@ standalone tracing and observability platform, Traceable created this open sourc
...
+git clone https://github.com/hypertrace/hypertrace.git
+cd hypertrace
+docker-compose -f docker/docker-compose.yml up
+```
+This will start all services required for Hypertrace. Once you see the service `Hypertrace-UI` start, you can visit Hypertrace UI at http://localhost:2020 . 
 
-### To start using Hypertrace
-- [Join the Hypertrace Workspace](https://www.hypertrace.org/get-started) on Slack to chat with other Hypertrace users.
-- You will be invited to a private channel where you can get the installer zip file along with instructions to install!
+If you have application instrumented to send traces to Zipkin or Jaeger, you are already covered with Hypertrace. Even if not you can run sample application using `docker-compose -f docker/docker-compose-zipkin-example.yml` or you can try Hypertrace with sample application by running
```
Comment (kotharironak) [inline README.md:22]: Can we move this `or you can try Hypertrace with sample application by running` as a completely separate line?
as e.g
```
If you want to run both Hypetrace and sample application together, you can use below command
```

File: kubernetes/clusters/dev/values.yaml [hunk 2]
```diff
@@ -0,0 +1,414 @@
+zookeeper:
```
Comment (kotharironak) [inline kubernetes/clusters/dev/values.yaml:1]: Can you have a comment in ` kubernetes/clusters/min/values.yaml` saying that it uses the default values?

File: README.md [hunk 3]
```diff
@@ -9,11 +9,37 @@ standalone tracing and observability platform, Traceable created this open sourc
...
+
+## Deploying with Kubernetes
+Please refer to [deployments](https://docs.hypertrace.org/deployments/) section in documentation which lists down steps for deploying Hypertrace on different Kubernetes flavors across different operating systems along with all major cloud providers. You can find the helm charts and installation script with more details [here](/kubernetes).
+
+## Community
+[Join the Hypertrace Workspace](https://www.hypertrace.org/get-started) on Slack to connect with other users, contributors and people behind Hypertrace. 
+
+## Documentation
+Check out [Hypertrace documentation](https://docs.hypertrace.org) to know more about Hypetrace features, architecture and more cool insights!
+- [Getting started](https://docs.hypertrace.org/getting-started/)
```
Comment (adriancole) [inline README.md:37]: I don't think the bullet points are needed. just the doc link section, especially as this one basically says what's already here. The one below on deployments also was mentioned above. Anyway, less bullets means less links to drift.

**Comment by adriancole** [comment]
implicit approve when comments addressed!

**Pull request merged**